src/app.module.ts

```diff
@@ -13,6 +14,6 @@ import { loadMongoConfig } from './config/helpers/load-mongo-config.helper';
     }),
   ],
   controllers: [AppController],
-  providers: [AppService],
+  providers: [{ provide: tokens.appService, useClass: AppService }],
```

Comment:
🎗️ As vitest uses esbuild to build JS, automatic dependency injection is not supported, so we need to manually inject each dependency